src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -37,12 +39,29 @@ object Run  {
     def toRunMap = params.map(p => p.name -> p.gcs).toMap.asJava
   }
 
+  private def dateTryToString(dateTry: Try[DateTime]): String = dateTry match {
+    case scala.util.Success(d: DateTime) => d.toString
+    case Failure(e: Throwable) => e.getClass.getName
+  }
+
   sealed trait RunStatus
   trait TerminalRunStatus extends RunStatus
-  final case class Initializing(created: DateTime) extends RunStatus
-  final case class Running(created: DateTime, started: DateTime) extends RunStatus
-  final case class Success(created: DateTime, started: DateTime, finished: DateTime) extends TerminalRunStatus
-  final case class Failed(created: DateTime, started: DateTime, finished: DateTime, errorCode: Int, errorMessage: String) extends TerminalRunStatus
+  final case class Initializing(created: Try[DateTime]) extends RunStatus {
+    override def toString = s"Initializing(created: ${dateTryToString(created)})"
+  }
+
+  final case class Running(created: Try[DateTime], started: Try[DateTime]) extends RunStatus {
+    override def toString = s"Running(created: ${dateTryToString(created)}, started: ${dateTryToString(started)})"
+  }
+
+  final case class Success(created: Try[DateTime], started: Try[DateTime], finished: Try[DateTime]) extends TerminalRunStatus {
+    override def toString = s"Success(created: ${dateTryToString(created)}, started: ${dateTryToString(started)}, finished: ${dateTryToString(finished)})"
+  }
+
+  final case class Failed(created: Try[DateTime], started: Try[DateTime], finished: Try[DateTime], errorCode: Int, errorMessage: String) extends TerminalRunStatus {
+    override def toString = s"Failed(created: ${dateTryToString(created)}, started: ${dateTryToString(started)}, finished: ${dateTryToString(finished)})"
+  }
+
```

Comment:
To restate a bit more clearly, it's because (normally) what JES returns is-a state that is initializing OR running OR success OR failed. And e.g. success has-a created AND started  AND finished date. Initializing only has-a created. Running has-a created AND started. 

The intention was to model the data after reality via an ADT, but again - the whole thing should be scrapped as it's all superfluous (we only care about started/finished, period) and JES can't be 100% trusted